bootstrap/daemon/sts.tf

```
@@ -39,13 +55,14 @@ resource "kubernetes_deployment_v1" "daemon" {
             value = "/fabric/daemon.toml"
           }
 
-          port {
-            container_port = local.port
+          volume_mount {
+            name       = "cache"
+            mount_path = "/var/cache"
           }
 
           volume_mount {
             name       = "config"
-            mount_path = "/fabric"
+            mount_path = "/fabric/config"
```

If we change this, we also have to change the `DAEMON_CONFIG` env var. I think this change is not necessary though
```suggestion
            mount_path = "/fabric"
```

---

yes, good caught